sig-contributor-experience/annual-report-2021.md

```diff
@@ -4,79 +4,105 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - [Migrating K-Dev to a managed kubernetes.io account](https://github.com/kubernetes/community/issues/5877)
+   - Steering Election with Elekto
```

Comment:
describe elekto a little more - created by lfx intern, what problem did it solve? etc